go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/crate-crypto/go-kzg-4844 v0.7.0
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
 	github.com/ethereum-optimism/go-ethereum-hdwallet v0.1.3
-	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20240222155908-ab073f6aa74f
+	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20240227110823-11c73f54ad9b
```

Comment:
need to update this once we put the op-sepolia-devnet-0 delta override into the superchain-registry pr.

Reply:
[2bb74bc](https://github.com/ethereum-optimism/optimism/pull/9642/commits/2bb74bc64f443adb536e6edb37bef81b4f078bb9)